public/js/leaderboard.js

```diff
@@ -75,13 +77,16 @@ var fetchLeaderboard = function() {
 fetchLeaderboard();
 
 var displayLeaderboard = function(studentLeaderList) {
-    
-
     getCurrentStudent();
     $('#leaderboardBody').html('');
     $('#criteriaName').html(boardType.displayName);
     podiumScore = studentLeaderList[0][boardType.name];
     $('#topScore').html(podiumScore + ((boardType === leaderboardTypes.ACCURACYBOARD) ? '%' : ''));
+
+    if (!leaderboardLimited) {
+        leaderboardLimit = 99999;
```

Comment:
I love how everyone is worried about the comments, but no one noticed that we have a fixed value here that makes no sense, and its back coding. But yeah lets spend the day fixing the comments, that will help

Reply:
I mean you did it sooo......